test/one_time_password_test.rb

```diff
@@ -22,10 +22,13 @@ def test_authenticate_with_otp
 
   def test_authenticate_with_otp_when_drift_is_allowed
```

Comment:
@hjinu I think would be better to add another model testing the intervals and add to test cases more instead of change the existing one